### DISCUSSION TITLE
v1.5.8-00

### DISCUSSION THREAD

**Discussion by gaborauth** [gaborauth/toolsboox-android#95] [Announcements]
- Added calendar marks (triangle for calendar and dots for notes)
- Added Spanish language, thank you Maira!
- Updated German language, thank you Thoms!
- Fixed month name on Boxed days calendar template
- Small bugfixes

<hr /><em>This discussion was created from the release <a href='https://github.com/gaborauth/toolsboox-android/releases/tag/v1.5.8-00'>v1.5.8-00</a>.</em>

**Comment by pjmysle** [comment]
I installed 1.5.8
How do use the calendar marks feature?

**Reply by gaborauth** [reply]
In the settings, you can sync the marks once, and from that the app manages the new marks, so, you can see marks on the calendar: a small triangle indicates a calendar entry with scribble, and a dot indicates a note page with scribble. :)
